Parse/src/main/java/com/parse/Parse.java

```diff
@@ -620,7 +627,13 @@ private static void initializeParseHttpClientsWithParseNetworkInterceptors() {
     interceptors.add(interceptor);
   }
 
-  // TODO(mengyan) Make public after we release the interceptor feature
+  /**
+   * Remove a given {@code ParseNetworkInterceptor}. You must invoke
+   * {@link #removeParseNetworkInterceptor(ParseNetworkInterceptor)}  before
+   * {@link #initialize(Context)} .
+   * @param interceptor
+   *          The {@code ParseNetworkInterceptor} to be removed.
```

Comment:
er.. based on the rule, this should be `{@link ParseNetworkInterceptor}`, right?